conf/drupal/config/filter.format.basic_html.yml

```diff
@@ -71,12 +71,9 @@ filters:
     weight: -11
     settings:
       title: false
-  media_embed:
```

Comment:
@arthurbaghdas Was removing media_embed here intentional?

Reply:
Yes, I've got this error https://app.circleci.com/pipelines/github/massgov/openmass/21693/workflows/5fd07c73-9595-4af0-a6d0-91be7fa32a1a/jobs/91652
So I assume we removed it a while ago, as I remember Moshe removed it, and this config was not touched since then.

Reply:
This conversation is resolved, we decided to implement other with other approach